src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEFieldSymbol.cs

```diff
@@ -413,7 +413,8 @@ public override Accessibility DeclaredAccessibility
                         break;
 
                     default:
-                        throw ExceptionUtilities.UnexpectedValue(_flags & FieldAttributes.FieldAccessMask);
+                        access = Accessibility.Private;
```

Comment:
This bug was meant to cover `public + private` accessibility flags.  This change expands it to all forms of accessibility not caught above.  Is that intentional?